Services/Administration/OJS.Services.Administration.Data/Implementations/SubmissionsDataService.cs

```diff
@@ -38,12 +39,12 @@ public IQueryable<Submission> GetAllByProblemAndParticipant(int problemId, int p
 
         public IQueryable<Submission> GetAllFromContestsByLecturer(string lecturerId)
             => this.DbSet
-                .Include(s => s.Problem!.ProblemGroup.Contest.LecturersInContests)
-                .Include(s => s.Problem!.ProblemGroup.Contest.Category!.LecturersInContestCategories)
+                .Include(s => s.Problem.ProblemGroup.Contest.LecturersInContests)
+                .Include(s => s.Problem.ProblemGroup.Contest.Category!.LecturersInContestCategories)
```

Comment:
Use `ThenInclude` instead of chaining in a single Include

Reply:
The following code has no impact on the created PR. Reverting my changes.